lua/astrocommunity/note-taking/obsidian-nvim/init.lua

```diff
@@ -1,17 +1,125 @@
 return {
   "epwalsh/obsidian.nvim",
-  cmd = {
-    "ObsdianBacklinks",
-    "ObsidianToday",
-    "ObsidianYesterday",
-    "ObsidianOpen",
-    "ObsidianNew",
-    "ObsidianSearch",
-    "ObsidianQuickSwitch",
-    "ObsidianLink",
-    "ObsidianLinkNew",
-    "ObsidianFollowLink",
-    "ObsidianTemplate",
+  lazy = true,
+  -- the obsidian vault in this default config  ~/obsidian-vault
+  event = { "BufReadPre  */obsidian-vault/*.md" },
+  -- If you want to use the home shortcut '~' here you need to call 'vim.fn.expand':
+  -- event = { "bufreadpre " .. vim.fn.expand "~" .. "/my-vault/**.md" },
+  dependencies = {
+    -- Required.
+    "nvim-lua/plenary.nvim",
+
+    -- Optional, for completion.
+    "hrsh7th/nvim-cmp",
+
+    -- Optional, for search and quick-switch functionality.
+    "nvim-telescope/telescope.nvim",
+
+    -- Optional, an alternative to telescope for search and quick-switch functionality.
+    -- "ibhagwan/fzf-lua"
+
+    -- Optional, another alternative to telescope for search and quick-switch functionality.
+    -- "junegunn/fzf",
+    -- "junegunn/fzf.vim"
+
+    -- Optional, alternative to nvim-treesitter for syntax highlighting.
+    -- "godlygeek/tabular",
+    -- "preservim/vim-markdown",
   },
-  opts = { completion = { nvim_cmp = true } },
+  opts = {
+    dir = "~/obsidian-vault", -- specify the vault location. no need to call 'vim.fn.expand' here
```

Comment:
I think it should use `vim.env.HOME `, to make it cross platform. I.e `vim.env.HOME`..`/obsidian-vault`

Reply:
Thanks. I made the change.